tests/e2e/run_all_tests.sh

```
@@ -28,15 +28,19 @@ echo "====================================${NC}"
 echo ""
 
 # Check if git-mind is built
-if [ ! -x "../../git-mind" ]; then
+# Determine binary location
+if [ -n "${GIT_MIND:-}" ] && [ -x "$GIT_MIND" ]; then
+    : # use provided GIT_MIND
+elif [ -x "../../git-mind" ]; then
+    export GIT_MIND="$(cd ../.. && pwd)/git-mind"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)